src/Mod/BIM/nativeifc/ifc_import.py

```diff
@@ -20,20 +20,19 @@
 # *                                                                         *
 # ***************************************************************************
 
-import importlib
 import os
 import time
 
 import FreeCAD
-from nativeifc import ifc_tools
-from nativeifc import ifc_psets
-from nativeifc import ifc_materials
-from nativeifc import ifc_layers
-from nativeifc import ifc_status
+
+from . import ifc_tools
+from . import ifc_psets
+from . import ifc_materials
+from . import ifc_layers
+from . import ifc_status
 
 if FreeCAD.GuiUp:
     import FreeCADGui
-    import Arch_rc
```

Comment:
Beware that, even if this is not used directly, it is still needed to load the Arch icons

Reply:
I'll revert that. What about unused `QtGui` twice in `ifc_import.py`?

Reply:
Thanks! If QtGui is not used anywhere, indeed it can be removed. It is probably a leftover from a previous state..